### PR TITLE
2024 feb updates

### PR DESCRIPTION
**Pull request by pbbgss**
@grahamison @dussaf This PR contains the updates we discussed in the pre-training meeting. I think you will need to download the [index.html file from this branch](https://github.com/moj-analytical-services/ggplotTraining/blob/2024-feb-updates/index.html) and open it in a browser to see the knitted slides. The presentation in the README link will update on merge.

### CODE REVIEW

**Comment by pbbgss** [comment]
I forgot to mention that when making the section 2 changes I noticed that the labels on [this](https://github.com/moj-analytical-services/ggplotTraining/blob/main/index.Rmd#L473-L479) chart were completely wrong (it is a plot of `cty` vs `hwy`) so I have updated them. Please could they get an extra thorough check for typos/spelling etc

File: index.Rmd [hunk 1]
```diff
@@ -283,6 +295,50 @@ The topics for the second part will include:
...
+`ggplot2::geom_smooth()` takes data points and returns a regression model with confidence intervals. For example, if we just replace `ggplot2::geom_point()` with `ggplot2::geom_smooth()`, we get a loess curve.  
+
+```{r, fig.height=3}
+ggplot2::ggplot(data = mpg, aes(x = displ, y = hwy)) +
+  ggplot2::geom_smooth()
+```
+
+---
+
+We can add the points as well, using the + operator; and have added a line break using `ggplot2::geom_line()` with a constant x value (this can be used for example to show a significant event in a time series). Note that the order of layers determines their order on the plot. As points are after the smooth line, they will be draw on top (and obscure) the line.
```
Comment (dussaf) [inline index.Rmd:331]: there is a typo at the end of the sentence drawn instead of draw

File: index.Rmd [hunk 2]
```diff
@@ -462,21 +473,20 @@ ggplot2::ggplot(data = mpg, aes(x = class, y = ..prop.., group = 1)) +
...
   ggplot2::ggplot(data = mpg, aes(x = cty, y = hwy)) +
   ggplot2::geom_jitter(colour = 'red') +
-  ggplot2::labs(title = "Highway fuel efficiency\number of cylinders", 
-                subtitle = "Note, the points are jittered", 
-                x = "Number of cylinders", 
-                y = "Fuel efficiency")
+  ggplot2::labs(title = "Fuel efficiency comparison", 
+                subtitle = "Note: the points are jittered", 
+                x = "City fuel efficiency", 
+                y = "Highway fuel efficienty")
```
Comment (dussaf) [inline index.Rmd:489]: There is a typo in efficiency

**Review by dussaf — APPROVED** [review]
Feels like a nice linear progression throughout the training.
Apart from the typo slide 41 (efficiency) and subsequent ones ( y axis label in graphs) it is all good.

**Pull request merged**